doc/user/content/transform-data/strategies-considerations.md

```diff
@@ -0,0 +1,159 @@
+  arriving data (e.g., records that arrive out of order due to network issues).
+
+- If using stacked views with temporal filters, avoid unnecessarily duplicating
+  the same temporal filter up the stack. Since Materialize maintains all future
+  retractions of the data that is currently within the temporal window (in
+  anticipation of when data will fall outside the window), removing unnecessary
+  temporal filters help reduce memory use.
+
+- [*Private preview*](/releases/previews/). Set an expiration for maintaining
+  future retractions. When using temporal filters, Materialize (in anticipation
+  of when data will fall outside the window) maintains all future retractions of
+  the data that is currently within the temporal window. Depending on your
+  dataset size and window size, you may want to configure Materialize to drop
+  expired retractions.
+
+### Static datasets
+
+Specifying [`REFRESH AT
+CREATION`](/sql/create-materialized-view/#refresh-strategies) for your
+materialized view effectively flags that view as static.
+
+If your data model consists of large datasets that join with smaller, static
+(i.e., does not change) datasets (such as a dimensional data model where your
+dimensions do not change), consider using materialized views with a [`REFRESH AT
+CREATION`](/sql/create-materialized-view/#refresh-strategies) specification for
+your static datasets. Joins with a static dataset may be able to reduce the
+amount of memory required for the other join inputs.
+
+### Hot/warm/cold data patterns
+
+If the freshness requirements of your workload vary such that some staleness is
+acceptable, consider using hot/warm/cold data patterns with differing refresh
+strategies.
+
+| Data Type       | Description                                                                                                                                               |
+|-----------------|-----------------------------------------------------------------------------------------------------------------------------------------------------------|
+| Hot data      | For data that requires immediate freshness, keep it in memory and use the default refresh strategy, which incrementally updates the data upon writes.     |
+| Warm/Cold data | For data that can support some degree of staleness, you can store this data in a materialized view that uses a less-frequent refresh strategy [(*Private preview*)](/sql/create-materialized-view/#refresh-strategies). |
+
+### Scheduled clusters
+
+For clusters that only contain [materialized views with scheduled refresh](/sql/create-materialized-view/#refresh-strategies), you can [schedule the cluster](/sql/create-cluster/#scheduling) to
+automatically turn on and off based on the refresh schedule.
+
+### Idiomatic Materialize SQL
+
+For various window functions, Materialize provides idiomatic Materialize SQL.
+Consider rewriting your queries using idiomatic SQL.
```

Comment:
will link to https://github.com/MaterializeInc/materialize/pull/29643/files#diff-2337ea1e435d8e6b252f7e4e31ad7ec4efff9379a77783b54da22f8e79755e2a once that's available.